src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let fixedRobot = {};
+
+  for (const key in robot) {
+    if (fixedRobot.hasOwnProperty(robot[key])) {
+      fixedRobot = null;
+
+      return fixedRobot;
+    } else {
+      fixedRobot[robot[key]] = key;
+    }
+  }
+
```

Comment:
<img width="1093" alt="Screenshot 2023-01-24 at 10 51 09" src="https://user-images.githubusercontent.com/94289038/214248484-103743ce-f7fb-4e31-9804-8a4bd62e9dce.png">
pay attention to checklist

Reply:
done